source/mainnet/smart-contracts/tutorials/wCCD/wCCD-introduction.rst

```diff
@@ -0,0 +1,91 @@
+.. _wCCD-introduction:
+
+=====================================
+Understanding the wCCD token protocol
+=====================================
+
+The native currency on the Concordium blockchain is CCD. When other tokens are
+built on the Concordium blockchain, they often use the recommended ``CIS-2``
+token standard. This has the advantage that other dApps (decentralized apps)
+can rely on some basic rules on how to interact with the ``CIS-2``
+tokens and on some basic rules on how the apps can retrieve events and data from the ``CIS-2`` tokens.
+The native currency CCD has a special intention in the Concordium
+blockchain network and does not comply with the ``CIS-2`` token standard.
+
+.. note::
+
+    The `CIS-2 standard <https://proposals.concordium.software/CIS/cis-2.html>`_
+    can represent fungible and non-fungible tokens.
+    It combines the Ethereum ``ERC20`` and ``ERC721`` standards with some modifications.
+    The `CIS-2 library <https://github.com/Concordium/concordium-rust-smart-contracts/blob/main/concordium-cis2/src/lib.rs>`_
+    is meant to be imported by ``CIS-2`` tokens.
+    Please explore the four ``CIS-2`` token standard implementation examples in the Concordium
+    smart contract repo:
+
+    - `wccd <https://github.com/Concordium/concordium-rust-smart-contracts/blob/main/examples/cis2-wccd/src/lib.rs>`_
+    - `upgradable wccd <https://github.com/Concordium/concordium-rust-smart-contracts/pull/128>`_
```

Comment:
Yes, I have written it down personally that I need to replace these links once the pull request 128 is merged.

Do you usually add an issue for it or write TODO (that is comment out) into the actual code to not forget about it?

Reply:
It depends a bit on the situation. Tasks are often preferred as they are the most visible. I usually just use todos for myself to remember things while developing and then resolve them before review.

In this scenario, a link to the PR is still of more value than no link, so I think we should keep it.

We often leave a review comment to point out todos, so that they aren't forgotten. We can keep this comment unresolved until the todo has been fixed (if we plan to merge the wCCD before this tutorial, that is ;) ).